### PR TITLE
Replace more `font-awesome` icons in `views/admin` area

### PR DESCRIPTION
**Pull request by mjankowski**
Followup on https://github.com/mastodon/mastodon/pull/29612 and https://github.com/mastodon/mastodon/pull/27780

This one handles the rest of the app/views/admin area that was not already handled. Like the previous one, I tried to use as close to a 1:1 mapping as there was, which sometimes had same name and other times did not. Also like last time, there is some small wiggle in the exact positioning.

Some before/after (not exhaustive, but representative):

Screenshot 2024-07-08 at 15 05 54 - save before
Screenshot 2024-07-08 at 15 07 59 - save after

<br />

Screenshot 2024-07-08 at 15 13 04 - warning before
Screenshot 2024-07-08 at 15 13 35 - warning after

<br />

Screenshot 2024-07-08 at 15 14 57 - hourglass before
Screenshot 2024-07-08 at 15 15 02 - hourglass after

<br />

Screenshot 2024-07-08 at 15 18 13 - retweet before
Screenshot 2024-07-08 at 15 18 32 - cycle after

<br />

Screenshot 2024-07-08 at 15 24 50 - server set before
Screenshot 2024-07-08 at 15 31 36 - server set after

### CODE REVIEW

File: app/views/admin/reports/_status.html.haml [hunk 1]
```diff
@@ -30,7 +30,7 @@
         %span.negative-hint= t('admin.statuses.deleted')
       ·
       - if status.reblog?
-        = fa_icon('retweet fw')
+        = material_symbol('cycle')
```
Comment (renchap) [inline app/views/admin/reports/_status.html.haml:33]: I think the one we use elsewhere for this is `repeat_active`

**Review by ClearlyClaire — DISMISSED** [review]
Not a fan of the `power`/`power_off` icons for the custom emoji but I can't really think of fitting icons. Very minor alignment issue in report interface with the new skip icon.

Otherwise this looks good to me.

**Comment by renchap** [comment]
Alignment looks weird, especially with the hourglass icon. I am not sure where it comes from, but it should be a simple CSS fix? Should this be something done for every material icon, to match FA's behaviour?

**Comment by vmstan** [comment]
> Not a fan of the `power`/`power_off` icons for the custom emoji but I can't really think of fitting icons. Very minor alignment issue in report interface with the new skip icon.
> 
> Otherwise this looks good to me.

I just noticed in the existing fa icons they're the same "power off" -- so at least this is an improvement.

**Comment by mjankowski** [comment]
> Alignment looks weird, especially with the hourglass icon. I am not sure where it comes from, but it should be a simple CSS fix? Should this be something done for every material icon, to match FA's behaviour?

Yeah, presumably there is slightly different padding in the svg between some of the material vs FA icons, and that leads to some of these different alignment issues.

My plan was to attempt to get them all replaced first in a "good enough, not outright broken" way, and then do another refinements pass wherever there are alignment/padding/whatever issues.

**Comment by mjankowski** [comment]
> Not a fan of the `power`/`power_off` icons for the custom emoji but I can't really think of fitting icons.

There is a "Power Settings New" icon in material that is basically the same as current FA power off, but it seems like we should correct the "enable and disable are using the same icon" thing while we're in here, and there's not really an "opposite" of that one. There are "toggle on" and "toggle off" in material which are almost meaningless but at least are different than each other ... could switch to that or something else instead of power/power_off.

**Comment by vmstan** [comment]
> > Not a fan of the `power`/`power_off` icons for the custom emoji but I can't really think of fitting icons.
> 
> There is a "Power Settings New" icon in material that is basically the same as current FA power off, but it seems like we should correct the "enable and disable are using the same icon" thing while we're in here, and there's not really an "opposite" of that one. There are "toggle on" and "toggle off" in material which are almost meaningless but at least are different than each other ... could switch to that or something else instead of power/power_off.

I think the toggles next to each other aren't really helpful if they don't actually toggle, as a single item individually. Just visually it doesn't make sense to me.

**Comment by mjankowski** [comment]
Switched to use a radio button checked/unchecked icon instead. Strikes me as marginally better than power options, and an improvement over the current same-icon state of things.

We're not there yet - but I want to avoid bikeshedding to exact perfection here; would strongly prefer to get a "good enough" pass at replacement, and then continue to refine/replace/restyle/etc in advance of release but after we fully remove FA.

**Comment by mjankowski** [comment]
Added a CSS change which improves the vertical align dynamic in the admin area table headers.

Before:

Screenshot 2024-07-22 at 10 14 11

After:

Screenshot 2024-07-22 at 10 13 58

**Review by Gargron — APPROVED** [review]

**Pull request merged**